website/docs/r/notification_channel.html.markdown

```diff
@@ -347,7 +347,7 @@ Additionally, the channel id isn't available via the UI, and you'd need to look
 That being said, importing is possible using -
 ```terraform import newrelic_notification_destination.foo <destination_id>```
 
-## Additional Information
+## Moving from Legacy Alert Channels to Notification Channels
 More details about the channels API can be found [here](https://docs.newrelic.com/docs/apis/nerdgraph/examples/nerdgraph-api-notifications-channels).
```

Comment:
Can you please restructure these lines in the following format:
- Restore **Additional Information** as a h2
- Move the "More details about the channels API can be found [here]...." line under the h2
- Add **Moving from Legacy Alert Channels to Notification Channels** as a h3 under the above line, which would contain all of the content we've added below this